tests/fetch/cached_xhr.cpp

```diff
@@ -24,7 +24,8 @@ void fetchFromIndexedDB()
     emscripten_fetch_close(fetch);
 
 #ifdef REPORT_RESULT
```

Comment:
what's the difference between `REPORT_RESULT` and what you've switched it to below? do we need another macro?

Reply:
The replacement JS has the same behavior as before (+ the fixes in this PR). The important thing is to have a single place that sends out results, so we can see if we send more than one. I guess we could make a C macro that just calls that JS, but it's about the same amount of typing?